source/manage/cloud-ip-filtering.rst

```diff
@@ -0,0 +1,80 @@
+Cloud IP Filtering
+========================
+
+.. include:: ../_static/badges/ent-cloud-only.rst
+   :start-after: :nosearch:
+
+
+IP Filtering is a powerful security feature that allows system administrators to control access to their workspace by defining approved IP ranges. Only users within these specified IP ranges can access the workspace, ensuring enhanced security for your workspace. IP Filtering requires a subscription to Mattermost Cloud Enterprise.  
+
+Accessing the IP Filtering Configuration:
+------------------------
+
+1. **Login as System Administrator:** Access the system console of your workspace, ensuring your user is a system administrator.
+
+2. **Navigate to Site Configuration:** Once logged in, navigate to the "Site Configuration" section within the admin console.
+
+3. **Access IP Filtering Settings:** Under "Site Configuration," select "IP Filtering" to access the IP Filtering settings.
+
+Understanding CIDR Notation:
+----------------------------
+
+
+CIDR (Classless Inter-Domain Routing) notation is used to specify a range of IP addresses. It consists of an IP address followed by a forward slash and a number indicating the network's prefix length. For example:
+
+   - `192.168.0.0/24` represents the IP range from `192.168.0.0` to `192.168.0.255`.
+   - The `/24` signifies that the first 24 bits are the network address, leaving 8 bits for host addresses.
+
+For a more in-depth explanation of CIDR notation, refer to `this article </https://aws.amazon.com/what-is/cidr/>`__.
```

Comment:
This feature is inherently technical in that it requires knowledge of CIDR notation. I tried to hit an even ground of explanation without rewriting wikipedia. Generally, if you don't know what CIDR notation is, you probably aren't the intended audience for this feature

Reply:
Very fair assumption to make. Mattermost product documentation focuses on Mattermost-centric details and points readers to external, third-party sources where appropriate. Nicely done.